release/mixer.sh

```diff
@@ -161,17 +165,41 @@ generate_mix() {
 
     build_update "${mix_ver}"
 
+    if [[ ${SIGN_MOM} == "true" ]]; then
+        sign_mom
+    fi
+
     build_deltas "${mix_ver}"
 
     echo -n "${mix_ver}" | sudo -E tee update/latest > /dev/null
 }
 
+sign_mom(){
+
+    if [ -z "${CUSTOM_MoM_CMD}" ]; then
+        warn "Skipping stage."
+        warn "No custom sign command for MoM is provided"
+    exit 0
+    fi
+
+    log "Signing MoM with custom command"
+    eval "${CUSTOM_MoM_CMD}"
+}
+
 # ==============================================================================
 # MAIN
 # ==============================================================================
 stage Mixer
 pushd "${MIXER_DIR}" > /dev/null
 
+#Check whether need to sign Manifest.MoM
+if [[ ${SIGN_MOM} == "false" ]]; then
```

Comment:
you can use CUSTOM_MoM_CMD to define everything.

If CUSTOM_MoM_CMD (a better name may be MOM_SIGN_CMD) you add the flag so mixer skip sign. (see comments) 

Also, if it is defined, you call it, otherwise no-op.

Final result would be:
if CUSTOM_SIGN_CMD defined:
    we use --no-signing to mixer calls
    we call the cmd to sign the manifest
otherwise code just run as usual